charts/fe-charts/templates/pdb.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.deployment.containers.env.pdb.enabled }}
```

Comment:
mending ini dipindah ke `.Values.pdb.enabled` aja gimana? since ini gak coupled ke `deployment.containers.env` IMO

sama nitip nambah default value juga di `[values.yaml](https://github.com/oyindonesia/fe-charts/blob/main/charts/fe-charts/values.yaml#L68)` sini ya

Reply:
bebas sih, format flaggingnya gua ngikut existing biar seragam tadinya
```
deployment:
  containers:
    env:
      enabled: true
      configmap:
        enabled: false
      secret:
        enabled: true
      pdb:
        enabled: false
```

okee default value otw

Reply:
addressed